packages/mapviewer/src/modules/infobox/components/FeatureAreaInfo.vue

```diff
@@ -62,7 +62,8 @@ const humanReadableArea = computed(() => {
             <span
                 class="align-middle"
                 data-cy="feature-area-information"
-            >{{ humanReadableArea }}<sup>2</sup></span>
+                >{{ humanReadableArea }}<sup>2</sup></span
+            >
```

Comment:
don't know if I like that...
We could add a linebreak after the span declaration IMHO